components/utility/button.js

```diff
@@ -0,0 +1,8 @@
+import { Button } from "./style"
+const Buttons = ({ children })=>{
```

Comment:
I don't thing there is any need to create a seperate component for button. How about you create and index.js file then all or most of the utilities should be written in the style.js

Reply:
If it's just a simple component that doesn't wrap anything except text let it be in the style.js file
But if it's a more complex component with multiple levels of nesting we'll make sure to create a folder for it for example that heading and the tab